### ISSUE TITLE
Possible scaling issue.

### ISSUE THREAD

**Issue by border999** [gnosygnu/xowa#758]
![image](https://user-images.githubusercontent.com/11824584/87269421-d062cc00-c468-11ea-9491-bed0248fa0fa.png)
The bottom is getting cut off so when it's attempting to display its progress it becomes guesswork about what it is saying sometimes.  My screens resolution is 3840x2160, and the version I am using is xowa_app_windows_64_v4.6.5.1911.

**Comment by border999** [comment]
also, if you could also find someway to better display the progress of imports and the like that would be helpful. I've had to keep my laptop running for a few days now because this wikipedia import won't end. Better knowing the time left on the import would be nice as well as a progress indicator that doesn't keep disappearing.

**Comment by gnosygnu** [comment]
Hi! Thanks for the interest in XOWA!

Thanks also for the screenshot as well as the specs on screen resolution. Unfortunately, this is a known issue with XOWA. There is a workaround detailed in https://github.com/gnosygnu/xowa/issues/24. The Options link has since changed to here: home/wiki/Special:XowaCfg?grp=xowa.app.startup

> also, if you could also find someway to better display the progress of imports and the like that would be helpful. I had to keep my laptop running for a few days because this wikipedia import took that long to run through all the necessary components. Knowing the time left on the import would be nice as well as a progress indicator that doesn't keep disappearing.

Yeah, it's a little hard as I need to estimate how many pages are in the dump as well as calculate how "fast" the machine is. I usually run it in fire-and-forget mode and it completes in a night. A few days is sort of long. Out of curiosity, what's the processor / memory on your laptop?

**Comment by border999** [comment]
![Screenshot (263)](https://user-images.githubusercontent.com/11824584/87373178-e59d3080-c524-11ea-8a8e-b68ecf54f6ba.png)

**Comment by gnosygnu** [comment]
Thanks for the screenshot. An i7 8700K 3.7 GHz with 32.0 GB RAM is pretty up to date.

That said, I forgot that the main reason for the delay is probably the search indexes. XOWA uses a PageRank algorithm to prioritize pages and that takes several hours for English Wikipedia. See http://xowa.org/home/wiki/Help/Features/Search/Build

**Comment by border999** [comment]
noted

**Comment by gnosygnu** [comment]
Yeah, sorry, but it is what it is. For what it's worth, the PageRank algo builds a fairly good search index. This means that typing `Eart` in the URL bar will pull the most linked article in Wikipedia and not what alphabetically comes first.

Also, out of curiosity, did the steps in #24 help with the progress bar? I know it's awkward but unfortunately it is the only workaround for now.

**Comment by border999** [comment]
it did work with the updated options link you provided here.

**Comment by gnosygnu** [comment]
Cool. Thanks for the update.

Let me know if there's anything else which should be done for this item. Again, the laptop resolution issue is an awkward hack, but unfortunately, there's no easier way for now.

Thanks!
